libs/sdk-ui-kit/src/SettingItem/typings.ts

```diff
@@ -0,0 +1,36 @@
+// (C) 2022 GoodData Corporation
+
+import { ReactElement } from "react";
+
+/**
+ * @internal
+ */
+export enum SettingItemActionType {
+    LinkButton = "LinkButton",
+    Button = "Button",
+    Switcher = "Switcher",
+}
+
+/**
+ * @internal
+ */
+export type ActionType = "LinkButton" | "Button" | "Switcher";
```

Comment:
These 2 interfaces (SettingItemActionType vs ActionType) look like the same thing, I think we should keep only the union type.

Reply:
Fixed. Thanks.